src/Microsoft.Sbom.Targets/Microsoft.Sbom.Targets.targets

```diff
@@ -58,23 +67,34 @@
       <Output TaskParameter="SbomPath" PropertyName="SbomPathResult" />
     </GenerateSbom>
     <Message Importance="High" Text="Task result: $(SbomPathResult)" />
+  </Target>
+
+  <!-- Specify the SBOM files to copy into the nuget package -->
+  <Target Name="CopySbomOutput" DependsOnTargets="GenerateSbomTarget">
+    <ItemGroup>
+      <!--Add manifest and SHA file from the GenerateSbomTask execution-->
+      <BuildOutputInPackage Condition=" '$(MSBuildRuntimeType)' == 'Core'"
+								  Include="$(SbomPathResult)\$(SbomSpecification)\manifest.spdx.json"
+								  TargetPath="_manifest/$(SbomSpecification)/manifest.spdx.json"/>
+      <BuildOutputInPackage Condition=" '$(MSBuildRuntimeType)' == 'Core'"
+								  Include="$(SbomPathResult)\$(SbomSpecification)\manifest.spdx.json.sha256"
+								  TargetPath="_manifest/$(SbomSpecification)/manifest.spdx.json.sha256"/>
 
-    <!-- Include the generated SBOM contents within the consumer's nuget package -->
-    <ItemGroup >
-      <Content Condition=" '$(MSBuildRuntimeType)' == 'Core'" Include="$(SbomPathResult)\**">
-        <Pack>true</Pack>
-        <PackagePath>_manifest</PackagePath>
-      </Content>
+      <!--Add manifest and SHA file from the SbomCLIToolTask execution from the default manifest path -->
+      <BuildOutputInPackage Condition=" '$(MSBuildRuntimeType)' == 'Full' And '$(SbomGenerationManifestDirPath)' == ''"
+								  Include="$(SbomGenerationBuildDropPath)\$(ManifestFolderName)\$(SbomSpecification)\manifest.spdx.json"
```

Comment:
Actually, this should be set to `ManifestDirPath` if it's not null or to $(SbomGenerationBuildDropPath)\$(ManifestFolderName) if it's null. We should return it from the ToolTask to make it consistent with the Task.

Reply:
Updated! Changed this to make the ToolTask set the SbomPath output variable we have